applications/models.py

```diff
@@ -294,6 +309,18 @@ class ApplicationRoundBasket(models.Model):
         verbose_name=_("Order number"), default=1, null=False, blank=True
     )
 
+    def get_application_events_in_basket(self, applicationevents):
```

Comment:
so the parameter of this excludes the given ApplicationEvents from the result?

Reply:
Yeah, through i renamed this better already but i'll give it more descriptive name.